packages/@okta/vuepress-site/docs/concepts/policies/index.md

```diff
@@ -12,54 +12,86 @@ Policies are evaluated when a request is made. Rules in the policies define perm
+
+* [Authentication policy](/docs/reference/api/policy/#authentication-policy): Determines the extra levels of authentication that you want performed before a user can access an application, such as enforcing factor requirements. Every app in your org has one authentication policy, and multiple apps can share a policy. Okta provides some preset policies with standard sign-on requirements, including a default policy automatically assigned to new apps. The default policy allows access with any two factors.
+
+  Create a policy specifically for the app or create a few policies and [share them](https://help.okta.com/okta_help.htm?type=oie&id=ext-share-auth-policy) across multiple apps. If you decide later to change an app’s sign-on requirements, you can modify its policy or switch to a different policy using the [Authentication Policies page](https://help.okta.com/okta_help.htm?type=oie&id=ext-create-auth-policy). See [Configure a Global Session Policy and an authentication policy](/docs/guides/configure-signon-policy/main/).
+
+### Password policy
+
+* [Password policy](/docs/reference/api/policy/#password-policy): Determines the requirements for a user's password length and complexity, as well as how often a user must change their password. Okta provides a default policy to enforce the use of strong passwords, but you can also create additional policies that are less or more restrictive and apply them to users based on group membership. This policy also governs the recovery operations that the user may perform, including change password, reset (forgot) password, and self-service password unlock.
+
+### Enrollment policies
 
-* [Authentication Policy](/docs/reference/api/policy/#authentication-policy): Determines the extra levels of authentication that you may want performed before an application can be accessed.
+* [Multifactor Authentication Policy](/docs/reference/api/policy/#multifactor-mfa-enrollment-policy): Controls how users enroll themselves in an authenticator. The policy controls which multifactor authentication (MFA) [methods](https://help.okta.com/okta_help.htm?type=oie&id=ext-about-authenticators) are available for a user, as well as when a user may enroll in a particular factor. Enable factors in your Okta org by creating a policy with one or more authenticators, and then assigning that policy to your app. See [Authenticators](https://developer.okta.com/docs/guides/authenticators-overview/main/) to learn how to increase the security of your app by requiring a user to verify their identity in more than one way.
```

Comment:
Do we "enroll authenticators"? Or "enroll IN authenticators"?

What's in this paragraph differs from the "Authenticators" docs.